app/javascript/dashboard/components/ChatList.vue

```diff
@@ -11,15 +11,24 @@ l<template>
           v-if="!hasAppliedFilters"
           @statusFilterChange="updateStatusType"
         />
-        <woot-button
-          v-else
-          size="small"
-          variant="clear"
-          color-scheme="alert"
-          @click="resetAndFetchData"
-        >
-          {{ $t('FILTER.CLEAR_BUTTON_LABEL') }}
-        </woot-button>
+        <div v-else>
+          <woot-button
```

Comment:
```suggestion
<woot-button
            size="tiny"
            variant="smooth"
            color-scheme="secondary"
            icon="save"
            @click="showAddCustomViewsModal = true"
         />
```

Could use self closing tags here